semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -53,6 +53,19 @@ def parse(span_kind_value):
         return kind_map.get(span_kind_value)
 
 
+class InstrumentKind(Enum):
+    Empty = 1
```

Comment:
Enum members should use ALL_CAPS

Reply:
👍  true, I've fixed it up. names of the instruments in the spec are CamelCase, so I had it like conveniently so I could map to `self.name`, but it wasn't too much trouble to write a mapping function to do that. 

Additionally, I've removed the asynchronous instruments as per https://github.com/open-telemetry/opentelemetry-specification/pull/2458